modules/@apostrophecms/doc-type/ui/apos/logic/AposDocContextMenu.js

```diff
@@ -245,9 +245,6 @@ export default {
       if (!this.context._id) {
         return false;
       }
-      if (!this.canEdit) {
-        return false;
-      }
```

Comment:
This check is done on backend now, even if some cases are not working rn.
eg. user can archive but cannot edit, the delete method perform a `findForEditing` that checks if user has the `edit` permission for that document.